### PR TITLE
Standardized README

### PR DESCRIPTION
**Pull request by RichardLitt**
Made this README compatible with github.com/RichardLitt/standard-readme. Added a contribute section, maintainers, a badge to the IPFS chatroom, and the github description, and a ToC

We should add an Install section. Can you help me with that?

See https://github.com/multiformats/multiformats/issues/13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,19 +60,19 @@ iex> Multihash.encode(0x20, :crypto.hash(:sha, "Hello"))
...
 iex> Multihash.encode(:sha1, :crypto.hash(:sha, "Hello"), 10)
 {:ok, <<17, 10, 247, 255, 158, 139, 123, 178, 224, 155, 112, 147>>}
 iex> Multihash.encode(:sha1, :crypto.hash(:sha, "Hello"), 30)
 {:error, "Invalid digest length"}
 ```
 
-## Decoding
+### Decoding
 
 Decode the provided multihash to %Multihash{code: , name: , length: , digest: }
```
Comment (zabirauf) [inline README.md]: Lets put it under the ` code markdown tag with the type information

`%Multihash{name: atom, code: integer, length: integer, digest: bitstring}`

File: README.md [hunk 2]
```diff
@@ -5,15 +5,38 @@ Multihash
...
+  - [Decoding](#decoding)
+    - [Examples](#examples-3)
+    - [Examples](#examples-4)
+- [Maintainers](#maintainers)
+- [Contribute](#contribute)
+- [License](#license)
+
+## Install
+
+TODO
```
Comment (zabirauf) [inline README.md]: This should have the following 

To use ex_multihash add to your `mix.exs` file:

```
defp deps do
  [
    {:ex_multihash, "~> 1.0"}
  ]
end
```

**Comment by zabirauf** [comment]
I've added a comment and what to add in Install section. Let me know if you want me to make those change as part of another commit.

**Comment by RichardLitt** [comment]
Added based on those comments. Should be good to go now.

**Comment by zabirauf** [comment]
Thanks Richard for updating the README.

**Comment by RichardLitt** [comment]
Thanks! No problem.

**Pull request merged**